.github/workflows/build-linux-arm64-installer.yml

```diff
@@ -110,6 +110,11 @@ jobs:
       run: |
         sh install.sh
 
+    - name: Setup Node 16.x
```

Comment:
This shouldn't be necessary - node should be preinstalled in the container we use (see line 17 of this file) `container: chianetwork/ubuntu-18.04-builder:latest`

That container comes from here https://github.com/Chia-Network/build-images/blob/main/ubuntu-18.04/Dockerfile#L34

Reply:
Yep. This should be removed. Making a new commit.